### ISSUE TITLE
The tokenizer class you load from this checkpoint is not the same type as the class this function is called from.

### ISSUE THREAD

**Issue by Xhai3208** [RollingPlain/SAGE_IVIF#9]
The tokenizer class you load from this checkpoint is not the same type as the class this function is called from. It may result in unexpected tokenization. 
The tokenizer class you load from this checkpoint is 'GPT2Tokenizer'. 
The class this function is called from is 'CLIPTokenizer'.
The tokenizer class you load from this checkpoint is not the same type as the class this function is called from. It may result in unexpected tokenization. 
The tokenizer class you load from this checkpoint is 'GPT2Tokenizer'. 
The class this function is called from is 'CLIPTokenizer'.
The tokenizer class you load from this checkpoint is not the same type as the class this function is called from. It may result in unexpected tokenization. 
The tokenizer class you load from this checkpoint is 'GPT2Tokenizer'. 
The class this function is called from is 'CLIPTokenizer'.
The tokenizer class you load from this checkpoint is not the same type as the class this function is called from. It may result in unexpected tokenization. 
The tokenizer class you load from this checkpoint is 'GPT2Tokenizer'. 
The class this function is called from is 'CLIPTokenizer'.

**Is the pre-trained model of my xdecoder downloaded incorrectly?**

**Comment by zhendebufapyq** [comment]
Thank you for your question. Please verify if the name of the pre-trained model you downloaded is `xdecoder_focall_last.pt`. If it is, then your pre-trained model was downloaded correctly.

**Comment by Xhai3208** [comment]
Thanks for your reply! I have another question, Is my training loss range normal: Teacher: step: 107, total_step: 7555, lr: 0.0004294680394738471, fuse_loss: 0.06470121443271637, grad loss: 0.04569612070918083, seg_loss: 0.9235958456993103, loss_student_new: 0.13060720264911652, loss: 1.1646004915237427.
